README.org

```diff
@@ -351,7 +351,7 @@ Above all, enjoy using Emacs. The community, that more than anything, makes Emac
 
    - [[https://www.emacswiki.org/emacs/DeleteSelectionMode][delete-selection-mode]] - =[built-in]= Auto delete selected region when typing, just like most of other modern text editors.
    - [[https://github.com/soutaro/hungry-delete.el][hungry-delete]] - Delete an entire block of whitespace at point.
-   - [[https://github.com/hrehfeld/emacs-smart-hungry-delete][smart-hungry-delete]] -  Delete whitespace between words, parenthesis and other delimiters in a (not very) smart way.
+   - [[https://github.com/hrehfeld/emacs-smart-hungry-delete][smart-hungry-delete]] -  Delete whitespace between words, parenthesis, and other delimiters in a (not very) smart way.
```

Comment:
The original description at https://github.com/hrehfeld/emacs-smart-hungry-delete contains the prior wording.

I'm not sure the Oxford Comma should/needs to stylistically be edited into this entire README.org file.